### DISCUSSION TITLE
Celebrate Hacktoberfest 2022 with Miru Web

### DISCUSSION THREAD

**Discussion by akhilgkrishnan** [saeloun/miru-web#678] [Announcements]
Welcome all the open-source developer for contributing to Miru Web

**Comment by Gismo1337** [comment]
Hey there. do you add some issues to work on for hacktoberfest contributors?

**Comment by akhilgkrishnan** [comment]
@Gismo1337 Yes, we are working on adding more issues on GitHub. You can join our [Discord](https://discord.gg/5XKQ8deg) channel for the discussions.

**Comment by supriya3105** [comment]
@Gismo1337 We have added good first issues on Github.

**Reply by Gismo1337** [reply]
@supriya3105 you can me assign to this. i would do this.
